python/cog/server/http.py

```diff
@@ -127,16 +127,9 @@ async def start_shutdown() -> Any:
 
     try:
         predictor_ref = get_predictor_ref(config, mode)
-        # use bundled schema if it exists
-        schema_module = None  # schema.create_schema_module() - breaks complex types, always use slow path for now
-        if schema_module is not None:
-            log.info("using bundled schema")
-            InputType = schema_module.Input
-            OutputType = schema_module.Output
-        else:
-            predictor = load_predictor_from_ref(predictor_ref)
-            InputType = get_input_type(predictor)
-            OutputType = get_output_type(predictor)
+        predictor = load_slim_predictor_from_ref(predictor_ref)
```

Comment:
Do we not want to fall back to the old way if this blows up?

Reply:
Ahhh, I see that this falls back itself to a full load.

Reply:
the fallback is inside load_slim_predictor_from_ref with try/catch. the load_full_predictor_from_file is the old(slow) way.